src/plugins/network.ts

```diff
@@ -207,159 +209,150 @@ export class NetworkManager {
    * Account for missing response received event and also adjust the
    * response received event based on when first byte event was recorded
    */
-  getResponseReceivedTime(
-    timing: Protocol.Network.Response['timing'],
-    responseReceivedTime: number
-  ) {
-    if (timing == null) {
-      return responseReceivedTime;
-    }
-    const startTime = timing.requestTime;
-    const headersReceivedTime = startTime + timing.receiveHeadersEnd / 1000;
-    if (
-      responseReceivedTime < 0 ||
-      responseReceivedTime > headersReceivedTime
-    ) {
-      responseReceivedTime = headersReceivedTime;
-    }
-    if (startTime > responseReceivedTime) {
-      responseReceivedTime = startTime;
-    }
+
+  stop() {
+    return this.results;
+  }
+}
+
+function getResponseReceivedTime(
+  timing: Protocol.Network.Response['timing'],
+  responseReceivedTime: number
+) {
+  if (timing == null) {
     return responseReceivedTime;
   }
+  const startTime = timing.requestTime;
+  const headersReceivedTime = startTime + timing.receiveHeadersEnd / 1000;
+  if (responseReceivedTime < 0 || responseReceivedTime > headersReceivedTime) {
+    responseReceivedTime = headersReceivedTime;
+  }
+  if (startTime > responseReceivedTime) {
+    responseReceivedTime = startTime;
+  }
+  return responseReceivedTime;
+}
+
+/**
+ * The timing calculations are based on the chrome devtools frontend
+ * https://github.com/ChromeDevTools/devtools-frontend/blob/7f5478d8ceb7586f23f4073ab5c2085dac1ec26a/front_end/network/RequestTimingView.js#L98-L193
```

Comment:
Can we add the license for this linked file to `NOTICE.txt` and reference that here?

Reply:
Done.